src/rtc/rtc.c

```diff
@@ -970,11 +970,8 @@ s32 sceRtcFormatRFC2822LocalTime(char *pszDateTime, const u64 *pUtc) {
     int oldK1 = pspShiftK1();
     s32 res = SCE_ERROR_INVALID_POINTER;
 
-    if((oldK1 & (int)pszDateTime) >= 0) {
-        if((oldK1 & (int)pszDateTime) >= 0) {
+    if(((oldK1 & (int)pszDateTime) >= 0) && ((oldK1 & (int)pszDateTime) >= 0))
```

Comment:
Haven't looked at the ASM, but shouldn't the condition be:

`if(((oldK1 & (int)pszDateTime) >= 0) && ((oldK1 & (int)pUtc) >= 0))`

just like in the sceRtcFormatRFC3339LocalTime function